pages/profile/change_password.js

```diff
@@ -0,0 +1,59 @@
+import Link from 'next/link';
+import { string } from 'prop-types';
+import withAuthSync from 'decorators/withAuthSync/withAuthSync';
+import { changePassword } from 'common/constants/api';
+import Head from 'components/head';
+import HeroBanner from 'components/HeroBanner/HeroBanner';
+import Content from 'components/Content/Content';
+import ChangePasswordForm from 'components/ChangePasswordForm/ChangePasswordForm';
+import styles from '../styles/password_reset.css';
+
+class ChangePassword extends React.Component {
+  static propTypes = {
```

Comment:
how would props get passed thru to these pages?

Reply:
Whoops, caught me copy-pasting